src/inference/src/ie_common.cpp

```diff
@@ -18,6 +18,7 @@
 #include "ie_parameter.hpp"
 #include "ngraph/opsets/opset.hpp"
```

Comment:
can you please avoid ngraph usage since you move to API 2.0?

Reply:
yes, CPU plugin API 2.0 should avoid ngraph usage, but ie_common.cpp is for legacy API which still be kept until 2024.0